staging/src/k8s.io/client-go/pkg/api/v1/types.go

```diff
@@ -2203,6 +2203,9 @@ type PodSpec struct {
 	// If not specified, the pod will be dispatched by default scheduler.
 	// +optional
 	SchedulerName string `json:"schedulername,omitempty" protobuf:"bytes,19,opt,name=schedulername"`
+	// If specified, the pod's tolerations.
+	// +optional
+	Tolerations []Toleration `json:"tolerations,omitempty" protobuf:"bytes,21,opt,name=tolerations"`
```

Comment:
AutomountServiceAccountToken, 21, is not part of staging/src/k8s.io/client-go/pkg/api/v1/types.go